tests/test_examples.py

```diff
@@ -54,10 +55,11 @@ def init_class_fixtures(cls):
             serialization='pickle',
         )
 
-        copy_market_data(WithTradingEnvironment.MARKET_DATA_DIR,
-                         cls.tmpdir.getpath('example_data/root'))
+        market_data = ('SPY_benchmark.csv', 'treasury_curves.csv')
+        for file in market_data:
+            ensure_file(cls.tmpdir.getpath('example_data/root/data/') + file)
```

Comment:
Just some nits now: Let's not name this variable `file`, so it doesn't mask the builtin. And let's use `os.path.join` instead of `+` to ensure the path separator is correct on windows and elsewhere - in fact, if we move `+ file` into the `getpath` call, it looks like it uses `os.path.join` for us under the hood. 

Otherwise LGTM!